media/js/base/send-to-device.js

```diff
@@ -90,17 +89,19 @@ if (typeof window.Mozilla === 'undefined') {
         // just show the email messaging after 5 seconds waiting.
         this.formTimeout = setTimeout(self.updateMessaging, 5000);
 
-        $.get('/country-code.json')
-            .done(function(data) {
+        window.fetch('/country-code.json')
```

Comment:
`fetch` isn't supported by Internet Explorer, so we may want to use the traditional `XMLHttpRequest`